test/Microsoft.AspNetCore.SignalR.Client.Tests/RpcConnectionFacts.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO.Pipelines;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Sockets.Client;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Microsoft.AspNetCore.SignalR.Client.Tests
+{
+    public class RpcConnectionFacts : IDisposable
+    {
+        TestServer testServer;
+        public RpcConnectionFacts()
+        {
+            var webHostBuilder = new WebHostBuilder().
+                ConfigureServices(services =>
+                {
+                    services.AddSignalR();
+                })
+                .Configure(app =>
+                {
+                    app.UseSignalR(routes =>
+                    {
+                        routes.MapHub<TestHub>("/hubs");
+                    });
+                });
+            testServer = new TestServer(webHostBuilder);
+        }
+
+        [Fact]
+        public async void CheckFixedMessage()
```

Comment:
All these async methods should return `Task` otherwise xUnit will not have a task to await and therefore may miss exceptions which will result in marking tests that failed as passing. `async void` should in general not be used outside async event handlers.